website/docs/reference/api-tokens-and-client-keys.mdx

```diff
@@ -55,6 +56,18 @@ Do **not** use admin tokens for:
 
 Support for scoped admin tokens with more fine-grained permissions is currently in the planning stage.
 
+
+### Project level access tokens
+
+All project members can see any `CLIENT` (server-side client token) and `FRONTEND` (client-side client token) respectively.
```

Comment:
Thanks! Looking at this, I think we could reword it to instead be about visibility. But I need you to help me out with the details here: how visible are tokens usually? I've got a suggestion below, but it needs to be modified based on what is the reality. I'd be happy to help rewrite this again, but I don't quite know how all the permissions around API tokens work these days.

Oh, and I also think this should be moved to before the admin tokens section.

```suggestion
### API token visibility

:::info project-level visibility

Project-level visibility and access to API tokens was introduced in Unleash 4.22.

:::


By default, only admin users can create API tokens, and only admins can see their values.

However, any [client](#client-tokens client tokens) and [front-end tokens](#front-end-tokens) that are applicable to a project, will also be visible to any members of that project that have the `READ_PROJECT_API_TOKEN` permission (all project members by default). 

Similarly, any project members with the `CREATE_PROJECT_API_TOKEN` permission can also create client and front-end tokens for that specific project ([how to create project API tokens](../how-to/how-to-create-project-api-tokens.mdx)).
```

Reply:
agreed and well done